extension.json

```diff
@@ -125,6 +125,9 @@
 		},
 		"BucketMaxDataPerPage": {
 			"value": 1000000
+		},
+		"BucketForceOldRepeatedQuery": {
+			"value": true
```

Comment:
Make this `false` by default. New users of Bucket shouldn't have this defaulted on. WG can set this to true ourselves in our config.